src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -533,9 +534,19 @@ public void WriteDatabaseRecord(DatabaseRecord databaseRecord, SmugglerProgressB
                 var currentDatabaseRecord = _database.ReadDatabaseRecord();
                 var tasks = new List<Task<(long Index, object Result)>>();
 
-                if (currentDatabaseRecord?.ConflictSolverConfig == null &&
-                    databaseRecord?.ConflictSolverConfig != null)
+                if (databaseRecord?.ConflictSolverConfig != null)
                 {
+                    if (currentDatabaseRecord?.ConflictSolverConfig != null)
+                    {
+                        foreach (var collection in currentDatabaseRecord.ConflictSolverConfig.ResolveByCollection)
+                        {
+                            if ((databaseRecord.ConflictSolverConfig.ResolveByCollection.ContainsKey(collection.Key)) == false)
```

Comment:
Do we need to do OridinalIgnoreCase here?

Reply:
No